### PR TITLE
Hotfix/add instance name in Deleting dialog

### PR DESCRIPTION
**Pull request by FredericEspiau**
## Add instance name in Deleting dialog

Internal reference: SCFRCA-922

### Description of the Change

Before: no indication of the deleted instance in the dialog
Now: the user can't get mixed up in the instance he or she is trying to delete

### CODE REVIEW

File: client/app/cloud/project/compute/infrastructure/virtualMachine/delete/cloud-project-compute-infrastructure-virtual-machine-delete.controller.js [hunk 1]
```diff
@@ -1,14 +1,12 @@
 "use strict";
 
 angular.module("managerApp")
-  .controller("CloudprojectcomputeinfrastructurevirtualmachinedeleteCtrl", function ($uibModalInstance, $stateParams, params, OvhApiCloudProjectIpFailover) {
-        var self = this,
-            serviceName = $stateParams.projectId;
-
-        var vmToDelete = params;
+    .controller("CloudprojectcomputeinfrastructurevirtualmachinedeleteCtrl", function ($uibModalInstance, params) {
```
Comment (antleblanc) [inline client/app/cloud/project/compute/infrastructure/virtualMachine/delete/cloud-project-compute-infrastructure-virtual-machine-delete.controller.js]: Could be written as a `class`? 💄

File: client/app/cloud/project/compute/infrastructure/virtualMachine/delete/cloud-project-compute-infrastructure-virtual-machine-delete.html [hunk 2]
```diff
@@ -20,7 +20,11 @@ <h2 class="oui-modal__title" data-translate="cpcivm_delete_title"></h2>
                     <li class="red" data-ng-repeat="ip in CloudprojectcomputeinfrastructurevirtualmachinedeleteCtrl.routedIpsFo track by ip.id">{{ ip.ip }}</li>
                 </ul>
             </div>
-            <p class="oui-modal__text"><strong data-translate="cpcivm_delete_confirm"></strong></p>
+            <p class="oui-modal__text">
```
Comment (antleblanc) [inline client/app/cloud/project/compute/infrastructure/virtualMachine/delete/cloud-project-compute-infrastructure-virtual-machine-delete.html]: What's about: 
```html
<p class="oui-modal__text font-weight-bold"
   data-translate="cpcivm_delete_confirm"
   data-translate-values="{ instanceName: $ctrl.vm.name }"></strong>
</p>
```

File: client/app/cloud/project/compute/infrastructure/virtualMachine/delete/cloud-project-compute-infrastructure-virtual-machine-delete.controller.js (hunk 1 above)
Comment (FredericEspiau) [reply client/app/cloud/project/compute/infrastructure/virtualMachine/delete/cloud-project-compute-infrastructure-virtual-machine-delete.controller.js]: It's a hotfix 😄. I'm all about refactoring to classes but not in a hotfix

File: client/app/cloud/project/compute/infrastructure/virtualMachine/delete/cloud-project-compute-infrastructure-virtual-machine-delete.html (hunk 2 above)
Comment (FredericEspiau) [reply client/app/cloud/project/compute/infrastructure/virtualMachine/delete/cloud-project-compute-infrastructure-virtual-machine-delete.html]: Works for me

**Pull request closed without merging**